grades/management/commands/populate_combined_final_grades.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management import BaseCommand
+
+from courses.models import Course
+from grades.api import update_or_create_combined_final_grade
+from grades.models import ProctoredExamGrade
+from micromasters.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """
+    Finds all passing exam grades for courses that have frozen runs
```

Comment:
Could you update this description and the help to indicate this generates the records in `CombinedFinalGrade`?